internal/bzlmod/go_deps.bzl

```diff
@@ -571,13 +656,13 @@ def _get_sum_from_module(path, module, sums):
         entry = (module.replace, module.raw_version)
 
     if entry not in sums:
-        fail("No sum for {}@{} found".format(path, module.raw_version))
+        fail("No sum for {}@{} found. You may need to run: bazel run @rules_go//go -- mod tidy".format(path, module.raw_version))
```

Comment:
TODO: Explore if we can also tell them what directory to run this in, as that won't be clear from the error message